File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/SettingsAct/settingFrManagerAct.java
package com.example.adminapp.SettingsAct;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.content.Intent;
import android.os.Bundle;

import com.example.adminapp.R;

public class settingFrManagerAct extends AppCompatActivity {

    public settingFrManagerAct() {}

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_setting_fr_manager);
        Intent intent = getIntent();
        String viewTag = intent.getStringExtra("view");
        loadFrag(viewTag);
    }

    private void loadFrag(String viewTag) {
        switch (viewTag){
            case "profile":
                FragmentManager fmp = getSupportFragmentManager();
                FragmentTransaction ftp = fmp.beginTransaction();
                ftp.add(R.id.frameLSettingAct,new profileFSettingAct());
                ftp.commit();
                break;
            case "signout":
                FragmentManager fms = getSupportFragmentManager();
                FragmentTransaction fts = fms.beginTransaction();
                fts.add(R.id.frameLSettingAct,new SignOutFSettingAct());
                fts.commit();
                break;
            case "tnc":
                FragmentManager fmtnc = getSupportFragmentManager();
                FragmentTransaction fttnc = fmtnc.beginTransaction();
                fttnc.add(R.id.frameLSettingAct,new TnCFSettingAct());
                fttnc.commit();
                break;
        }
    }
}

File: KnowledgeSpaceApk/adminapp/src/main/java/com/example/adminapp/SettingsAct/settingFrManagerAct.kt
package com.example.adminapp.SettingsAct

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.adminapp.R

class settingFrManagerAct : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_setting_fr_manager)
        val intent = intent
        val viewTag = intent.getStringExtra("view")
        loadFrag(viewTag)
    }

    private fun loadFrag(viewTag: String?) {
        when (viewTag) {
            "profile" -> {
                val fmp = supportFragmentManager
                val ftp = fmp.beginTransaction()
                ftp.add(R.id.frameLSettingAct, profileFSettingAct())
                ftp.commit()
            }

            "signout" -> {
                val fms = supportFragmentManager
                val fts = fms.beginTransaction()
                fts.add(R.id.frameLSettingAct, SignOutFSettingAct())
                fts.commit()
            }

            "tnc" -> {
                val fmtnc = supportFragmentManager
                val fttnc = fmtnc.beginTransaction()
                fttnc.add(R.id.frameLSettingAct, TnCFSettingAct())
                fttnc.commit()
            }
        }
    }
}